wavefront/server/modules/voice_agents_module/voice_agents_module/models/tool_schemas.py

```
@@ -0,0 +1,294 @@
+    def validate_method(cls, v):
+        allowed_methods = ['GET', 'POST', 'PUT', 'PATCH', 'DELETE']
+        if v.upper() not in allowed_methods:
+            raise ValueError(f'Method must be one of {allowed_methods}')
+        return v.upper()
+
+    @field_validator('url')
+    @classmethod
+    def validate_url(cls, v):
+        # Basic URL validation
+        if not v.startswith(('http://', 'https://')):
+            raise ValueError('URL must start with http:// or https://')
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)